crates/block-producer/src/test_utils/proven_tx.rs

```diff
@@ -108,7 +108,7 @@ impl MockProvenTxBuilder {
             .map(|index| {
                 let nullifier = Word::from([ONE, ONE, ONE, Felt::new(index)]);
 
-                Nullifier::from(nullifier)
+                Nullifier::new_unchecked(nullifier)
```

Comment:
I don't really love using `new_unchecked()` as it makes it feel like we are doing something dangerous, while in fact any word can be converted to a nullifier.

@drahnr, @PhilippGackstatter - I believe this resulted from the word-wrapper macro changes. Is there a better way to do this?

Reply:
Good point. I think we can revive `From<Word> for Nullifier`. Opened https://github.com/0xMiden/miden-base/pull/2134.

Reply:
Though maybe the wording `new_unchecked` is incorrect, and could be something like `from_raw`. But definitely no `From<Word>` should exist imo.